insights/specs/datasources/leapp.py

```diff
@@ -96,16 +91,18 @@ def migration_results(broker):
                 results = []
                 for activity in json_report.get('activities', []):
                     ret = {}
-                    for key, sub_keys in valid_keys.items():
+                    for key, filter_fn in valid_keys.items():
                         if key not in activity:
                             continue
                         val = activity.get(key)
-                        if not sub_keys:
+                        if filter_fn is None:
```

Comment:
@andywaltlova  - The current code can miss env key when its value is []. Update this line as if filter_fn is None or not val: can pass the test.

Reply:
Thank you! Saved me the investigation :)